src/pages/topics.tsx

```diff
@@ -0,0 +1,226 @@
+                        }}
+                        variant="heading5"
+                    >
+                        Technologies
+                    </TypographyScale>
+                    <ul sx={ULStyles}>
+                        {getTechnologies?.map(({ tagName, slug }) => (
+                            <li key={tagName}>
+                                <TypographyScale variant="body1">
+                                    <a href={getURLPath(slug)}>{tagName}</a>
+                                </TypographyScale>
+                            </li>
+                        ))}
+                    </ul>
+                    <TypographyScale
+                        sx={{ marginBottom: 0 }}
+                        variant="heading5"
+                    >
+                        Products
+                    </TypographyScale>
+                    {[
+                        Atlas,
+                        Compass,
+                        Connectors,
+                        MongoDB,
+                        OpsManager,
+                        Realm,
+                    ].map(({ slug, tagName, topics }) => (
+                        <div key={tagName}>
+                            <TypographyScale
+                                sx={{
+                                    fontWeight: '700',
+                                    marginTop: 'inc50',
+                                    marginBottom: [
+                                        'inc30',
+                                        'inc30',
+                                        'inc30',
+                                        'inc30',
+                                    ],
+                                }}
+                                variant="heading6"
+                            >
+                                <a href={getURLPath(slug)}>{tagName}</a>
+                            </TypographyScale>
+                            {!!topics.length && (
+                                <ul sx={{ ...ULStyles, marginBottom: 'inc90' }}>
+                                    {topics?.map(({ title, href }) => (
+                                        <li key={title}>
+                                            <TypographyScale variant="body1">
+                                                <a href={getURLPath(href)}>
+                                                    {title}
+                                                </a>
+                                            </TypographyScale>
+                                        </li>
+                                    ))}
+                                </ul>
+                            )}
+                        </div>
+                    ))}
+                </div>
+            </GridLayout>
+        </>
+    );
+};
+
+export default Topic;
+
+export const getStaticProps: GetStaticProps = async ({}) => {
+    const client = clientFactory('ApolloREST', process.env.STRAPI_URL);
```

Comment:
Don't think we need to bother instantiating this here.

Reply:
Will remove in separate PR